src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicApi.cs

```diff
@@ -147,6 +161,23 @@ static MsQuicApi()
         private static bool IsWindowsVersionSupported() => OperatingSystem.IsWindowsVersionAtLeast(MinWindowsVersion.Major,
             MinWindowsVersion.Minor, MinWindowsVersion.Build, MinWindowsVersion.Revision);
 
+        private static bool IsTls1_3Disabled()
+        {
+#if WINDOWS
+            const string SChannelTLS1_3RegKey = @"SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3";
```

Comment:
I don't think this is correct. It should be `Enabled` in `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client` and `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Server` AFAIK

Reply:
I noticed this after I started testing a different implementation locally.

Are there any implications as to how this should be handled due to client and server being independently disableable?

Reply:
since the Schannel control is separate most accurate would be to trace it also separately and distinguish between in and out connections e.g client vs server. But since something can be disabled while the process is running the accuracy is already not perfect. I feel it may be OK track it together e.g. set TLs13Disable if either side is disabled and 
```c#
if (errorCode == 0x80090331 && Tls13MayBeDisabled)
{
   throw NiceDisabledException()
}
else
{
   throw lowLevelOS/Quick
}
```